### PR TITLE
Implemented missing functionality for multiple overhead buttons and switches

### PR DESCRIPTION
**Pull request by wpine215**
**Summary of Changes**
- Implemented functionality for hydraulic overhead panel
- Implemented functionality for packs/cond overhead panel
- Implemented functionality for ventilation overhead panel
- Implemented functionality for manual engine start buttons
- Implemented functionality for EMER cabin call
- Implemented functionality of both overhead and pedestal door buttons
- Fixed IDG1 and IDG2 disconnect buttons to display FAULT message for remainder of flight if pushed
- Simvars are included and documented for all mentioned buttons/switches

**Screenshots (if necessary)**
FlightSimulator_6kLcUwRThx

* Keep in mind fault lights are functional and have simvars associated with them as well, but are not visible in the above screenshot

**References**
Provided by komp:
annunciator

*Keep in mind some of the provided emissive button textures are incorrect in the Asobo model, so they do not perfectly adhere to the above reference

**Additional context**

- Non-continuous motion of the temperature selector knobs is a known issue. Must be fixed in future PR.
- HYD RAT MAN ON button does not have an emissive texture.

Discord username (if different from GitHub): Iceman#5540

### CODE REVIEW

File: A32NX/ModelBehaviorDefs/Asobo/Airliner/Airbus.xml [hunk 1]
```diff
@@ -1177,17 +1174,6 @@
 			<KNOB_NUM_STATE>7</KNOB_NUM_STATE>
```
Comment (ThePilot916) [inline A32NX/ModelBehaviorDefs/Asobo/Airliner/Airbus.xml:1174]: Can you make these temperature knobs have continuous mode instead of 7 states?

**Comment by Carter633** [comment]
Name: Carter633 #1436
Date of testing: Current date 01/10/2020
Version of the sim: 1.9.3.0
PR Tested: #1057

Testing technique: Spawn cold and dark, check all functions/lights against above images
Steps you've made to test the Pull Request:
1. Spawned, cold/dark. Did ANN LT TST to check all lights.
2. All buttons etc noted/highlighted above were pressed and checked
3. No issues found

Overall Rating: Good
Conclusions of the testing: Seems fine!

**Comment by Pleasure0102** [comment]
looks good :) all worked for me

File: A32NX/ModelBehaviorDefs/Asobo/Airliner/Airbus.xml (hunk 1 above)
Comment (wpine215) [reply A32NX/ModelBehaviorDefs/Asobo/Airliner/Airbus.xml:1174]: I tried using the continuous knob template recycled from the light knobs but just couldn't get them to work for some reason. Maybe someone else will have better luck, or I'll get to them in another PR.

**Review by ThePilot916 — APPROVED** [review]

**Review by wuperguy — APPROVED** [review]

File: .github/CHANGELOG.md [hunk 2]
```diff
@@ -23,3 +23,4 @@
 9. [ND] Add DME distances, VOR/ADF needles and functioning ADF2 - @blitzcaster (bltzcstr)
 10. [OVHD] Fixed Battery Indicator Colour - @nathaninnes (Nathan Innes)
 11. [MISC] Removed Fuel Patch from MSFS Update 1.8.3 - @nathaninnes (Nathan Innes)
+12. [OVHD] Implemented missing functionality for multiple overhead buttons and switches - @wpine215 (Iceman)
```
Comment (NathanInnes) [inline .github/CHANGELOG.md:26]: Change to current month

Comment (wpine215) [reply .github/CHANGELOG.md]: Fixed.

Comment (lousybyte) [reply .github/CHANGELOG.md]: Can you please merge master and update the changelog location again, preview doesn't look right.

**Pull request merged**